### ISSUE TITLE
0.5.3: createprocess failed after installing

### ISSUE THREAD

**Issue by CMTriX** [Timthreetwelve/WUView#7]
Installing new version 0.5.3 fails when 'Launce WUv' is checked.

CreateProcess failed code 740. 
Unable to execute file WUVIEUW.EXE

system W11 22h2

**Comment by CMTriX** [comment]
When running setup as admin, there is no error, and i can start.Without admin start no log file is created.

With run as admin, there is a log created.

```
2023/05/08 17:56:01 INFO   WUView (Windows Update Viewer) 0.5.3.0 is starting up
2023/05/08 17:56:01 INFO   WUView Copyright © 2023 Tim Kennedy
2023/05/08 17:56:01 DEBUG  WUView Build date: woensdag 5 juli 2023 21:59 (UTC)
2023/05/08 17:56:01 DEBUG  WUView Commit ID: ab3bded 
2023/05/08 17:56:01 DEBUG  Operating System version: Microsoft Windows 10.0.22621
2023/05/08 17:56:01 DEBUG  .NET version:  6.0.16
2023/05/08 17:56:01 DEBUG  Read 1 exclude record from disk in 26,02 milliseconds
2023/05/08 17:56:01 INFO   Excluding updates containing: "Defender"
2023/05/08 17:56:01 DEBUG  Read 16 Setup event log records in 11,95 milliseconds
2023/05/08 17:56:02 DEBUG  Read 83 Windows Update records in 186,53 milliseconds
2023/05/08 17:56:02 WARN   KB:KB5007651  Date: 7-5-2023 15:21:09       HResult: -2145099769  Operation: uoInstallation  UpdateID: a36e5357-13f8-403e-ac5b-c0e31fd6c907
2023/05/08 17:56:02 DEBUG  Extracting KB numbers from update titles took 0,98 milliseconds
2023/05/08 17:56:02 DEBUG  Building WUpdate object took 9,19 milliseconds
2023/05/08 17:56:02 DEBUG  Building the list of updates took 16,75 milliseconds
2023/05/08 17:56:02 DEBUG  Removing excluded items took 0,99 milliseconds
2023/05/08 17:56:18 DEBUG  Checking GitHub for latest release.
2023/05/08 17:56:18 DEBUG  Latest version is 0.5.3 released on 7-5-2023
2023/05/08 17:56:18 DEBUG  No newer releases were found.
2023/05/08 17:56:21 INFO   WUView is shutting down.  Elapsed time: 0:00:20.21
2023/05/08 17:56:21
```

**Comment by CMTriX** [comment]
Installed first time on other computer. Checkbox for run after install checked, programm opens, and closing within second. No error. Starting programma again, it stays open.

**Comment by Timthreetwelve** [comment]
Thanks so much for sending the second log. I was able to reproduce the error and determined that it's crashing on when it creates the WUViewExcludes.json file. The file is created but can't be accessed. Subsequent executions of WUView won't encounter this problem. Should be an easy fix. Thanks again!

Tim

**Comment by Timthreetwelve** [comment]
This fix will be in release 0.5.4 which should publish tomorrow.
